packages/tasit-action/src/contract/Subscription.js

```diff
@@ -29,6 +36,14 @@ export class Subscription {
     this.#eventListeners.delete(eventName);
   };
 
+  getEventsTimeout = () => {
+    return this.#timeout;
+  };
+
+  setEventsTimeout = timeout => {
```

Comment:
Should we call it something other than events to indicate that it can relate to confirmations too and not just “contract events”?

Reply:
Ok, `setTimeout` sounds good?